### ISSUE TITLE
onChange hook does not update

### ISSUE THREAD

**Issue by another-novelty** [haoxins/react-flatpickr#193]
I have a simple controlled component which basically wraps a flatpickr:

```ts
const ControlledPicker: React.FC<{ value: Date, onChange: (dates: Date[]) => void }> = ({value, onChange}) => {
    return <Flatpickr
        value={value}
        onChange={onChange}
    />
}
```

If the external `onChange` changes,, then the function attached to the flatpickr instance is not updated. My current workaround is to manually change it with a `useEffect` hook:

```ts
const ControlledPicker: React.FC<{ value: Date, onChange: (dates: Date[]) => void }> = ({value, onChange}) => {
    const flatpickrRef = useRef<Flatpickr>(null);
    useEffect(() => {
        if (flatpickrRef.current) {
            flatpickrRef.current.flatpickr.config.onChange = [onChange];
        }
    }, [onChange]);

    return <Flatpickr
        value={value}
        onChange={onChange}
        ref={flatpickrRef}
    />
}
```

My current installed version is 3.10.12

**Comment by jacobmischka** [comment]
Thank you for the report! This should be fixed in 3.10.13. Let me know if you still see issues!

**Comment by another-novelty** [comment]
Yes! That worked. Thank you!
